src/algebra/order/hom/basic.lean

```diff
@@ -19,13 +19,17 @@ This file defines hom classes for common properties at the intersection of order
 * `subadditive_hom_class`: Homs are subadditive: `∀ f a b, f (a + b) ≤ f a + f b`
 * `submultiplicative_hom_class`: Homs are submultiplicative: `∀ f a b, f (a * b) ≤ f a * f b`
 * `mul_le_add_hom_class`: `∀ f a b, f (a * b) ≤ f a + f b`
+
+## TODO
+
+Finitary versions of the current lemmas.
```

Comment:
@YaelDillies, I have some lemmas about the nonarchimedean property applied to sums of finsets and multisets (not using `nonarchimedean_hom_class` exactly, but I will try to adapt them after this is merged).

Reply:
Great, thanks! I wasn't very willing to write them myself `nonarchimedean_hom_class` because I didn't know how to best handle the nullary case.